### DISCUSSION TITLE
Nominate @thitch97 as a contributor

### DISCUSSION THREAD

**Discussion by sophiewigmore** [paketo-buildpacks/feedback#108] [Dependencies Team]
I am nominating @thitch97 as a contributor on the Dependencies Contributor's team. Tim has made a number of contributions to dependencies in the last few months, so he should be able to make regular contributions to our repositories.

@paketo-buildpacks/dependencies-maintainers can I get a 👍 / 👎 ?

**Comment by ryanmoran** [comment]
With a super-majority vote to affirm, @thitch97 will be promoted to Dependencies Contributor. Congrats!
